src/rebar_file_utils.erl

```diff
@@ -198,17 +218,18 @@ delete_each_dir_win32([Dir | Rest]) ->
     delete_each_dir_win32(Rest).
 
 xcopy_win32(Source,Dest)->
-    {ok, R} = rebar_utils:sh(
-                ?FMT("xcopy \"~s\" \"~s\" /q /y /e 2> nul",
-                     [filename:nativename(Source), filename:nativename(Dest)]),
-                [{use_stdout, false}, return_on_error]),
-    case length(R) > 0 of
-        %% when xcopy fails, stdout is empty and and error message is printed
-        %% to stderr (which is redirected to nul)
-        true -> ok;
-        false ->
+    %% "xcopy \"~s\" \"~s\" /q /y /e 2> nul", Chanegd to robocopy to
+    %% handle long names. May have issues with older windows.
+    {_, {Rc, _}} = rebar_utils:sh(
```

Comment:
Maybe move it to the case below? For robocopy? 

So it does something like:
Case res of
 {_,{rc, -}} when rc < 9; rc =:= 16 ->... 
{ok, []} -> ok... 

I'm at work and writing on a mobile. :)